### ISSUE TITLE
support better sorting of flattened columnames

### ISSUE THREAD

**Issue by djdeejay** [amirziai/flatten#23]
Hi, i would like to request adding an formatstring for bulding new keynames as it happens stuff like this:
key_10_name, key_11_name, key_1_name, key_2_name ....
with 
```
 for index, item in enumerate(object_):
                _flatten(item, _construct_key(key, separator, '{:02d}'.format(index)))
```
the keys would look like this 
key_01, key_02, ...., , key_10_name, key_11_name

regards
djdeejay

**Comment by amirziai** [comment]
@djdeejay Makes sense. Since you've done the work do you want to raise a PR and show that it works with a test?
